daemon/inspect_fs_unix_fstab.ml

```diff
@@ -542,6 +542,32 @@ and resolve_dev_mapper spec default =
   (* Ignore devices that don't exist. (RHBZ#811872) *)
   | Unix.Unix_error (Unix.ENOENT, _, _) -> default
 
+and resolve_crypttab spec aug =
+  let augpath = sprintf "/files/etc/crypttab/*[target='%s']/device"
+    (Filename.basename spec) in
+  match aug_get_noerrors aug augpath with
+  | None -> None
+  | Some device ->
+    (if verbose() then eprintf "mapped to crypttab device=%s\n%!" device;
```

Comment:
No need for open paren `(` at the start of this line.

Reply:
done both